base/gfspclient/gater.go

```diff
@@ -26,6 +27,10 @@ const (
 	GnfdIntegrityHashHeader = "X-Gnfd-Integrity-Hash"
 	// GnfdIntegrityHashSignatureHeader defines integrity hash signature, which is used by receiver
 	GnfdIntegrityHashSignatureHeader = "X-Gnfd-Integrity-Hash-Signature"
+	//RecoveryObjectPiecePath defines recovery-object path style
+	RecoveryObjectPiecePath = "/greenfield/recovery/v1/get-piece"
```

Comment:
This APi is only used in SP internally?

Reply:
Why defines a path in this file?

Reply:
yes, used in SP internally,  similar with ReplicateObjectPiecePath

Reply:
RecoveryObjectPiecePath can be moved to another file